rapidxml.hpp

```diff
@@ -657,6 +657,7 @@ namespace rapidxml
         xml_base()
             : m_name(0)
             , m_value(0)
+            , m_value_size(0)
```

Comment:
The original code deliberately left m_value_size uninitialized (it's only accessed if m_value != nullptr)